web/src/components/AppBar/AppBar.vue

```diff
@@ -49,16 +49,22 @@
 
     <v-spacer />
 
-    <span v-if="!loggedIn" class="mr-1">
-      Want to create your own datasets?
-    </span>
-
     <template v-if="loggedIn">
-      <UserMenu
-        v-if="loggedIn"
-      />
+      <v-btn
+        :to="{ name: 'createDandiset' }"
+        exact
```

Comment:
This usage of `exact` is a smart idea. Until we have a reason to want the default regex matching, it's probably safer to always do it exactly. I'll file an issue.